src/scripts/main.js

```diff
@@ -358,3 +358,22 @@ const people = [
 console.log(people); // you can remove it
 
 // write your code here
+const table = document.querySelector('.dashboard');
+
+people.forEach((person) => {
+  const age = person.died - person.born;
+  const centery = Math.ceil(person.died / 100);
+  const gender = person.sex === 'm' ? 'Male' : 'Female';
+
+  const row = document.createElement('tr');
+
+  row.innerHTML = `
+    <td>${person.name}</td>
+    <td>${gender}</td>
+    <td>${person.born}</td>
+    <td>${person.died}</td>
+    <td>${age}</td>
+    <td>${centery}</td>
```

Comment:
Ensure that the variable name `centery` is corrected to `century` here as well to maintain consistency and accuracy.